### PR TITLE
[x/programs/cmd/simulator] Improve README.md

### PR DESCRIPTION
**Pull request by youngkashew**
## Description
* Fix commands to run test programs
* Fix typos
* Format syntax for consistency

### CODE REVIEW

File: x/programs/cmd/simulator/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # Program VM simulator (work in progress)
 
 The `Simulator` is currently a work-in-progress.
- It is a cli-tool built in Go, but meant to be used from the `wasmlanche-sdk` as a way to run automated tests on your HyperSDK wasm programs. For relatively up-to-date documentation, run `cargo doc --no-deps -p simulator --open`. Create a `Client`, create a `Plan`, then call `client.execute(plan)`, to execute the plan.
 
-And please, if you see any inconsistencies in the `README.md`` here, [open a PR](https://github.com/ava-labs/hypersdk/edit/main/x/programs/cmd/simulator/README.md)!
+It is a cli-tool built in Go, but meant to be used from the `wasmlanche-sdk` as a way to run automated tests on your HyperSDK Wasm programs. For relatively up-to-date documentation, run `cargo doc --no-deps -p simulator --open`. Create a `Client`, create a `Plan`, then call `client.execute(plan)`, to execute the plan.
```
Comment (wlawt) [inline x/programs/cmd/simulator/README.md:5]:
```suggestion
It is a CLI-tool built in Go, but meant to be used from the `wasmlanche-sdk` as a way to run automated tests on your HyperSDK Wasm programs. For relatively up-to-date documentation, run `cargo doc --no-deps -p simulator --open`. Create a `Client`, create a `Plan`, then call `client.execute(plan)`, to execute the plan.
```

File: x/programs/cmd/simulator/README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+
+And please, if you see any inconsistencies in the `README.md` here, [open a PR](https://github.com/ava-labs/hypersdk/edit/main/x/programs/cmd/simulator/README.md)!
 
 #### Note
 
-Are we calling a Go based-cli from Rust? Yes. The Go-cli re-uses primitives from the HyperSDK, but we wanted to wrap that code in a Rust client to give a seamless experience testing.
+Are we calling a Go-based CLI from Rust? Yes. The Go-cli re-uses primitives from the HyperSDK, but we wanted to wrap that code in a Rust client to give a seamless experience testing.
```
Comment (wlawt) [inline x/programs/cmd/simulator/README.md:11]:
```suggestion
Are we calling a Go-based CLI from Rust? Yes. The Go-CLI re-uses primitives from the HyperSDK, but we wanted to wrap that code in a Rust client to give a seamless experience testing.
```

**Review by wlawt — COMMENTED** [review]
lgtm! but not sure why the unit test is failing : /

**Comment by youngkashew** [comment]
@wlawt I pulled the latest upstream changes to see if that solves the failing unit tests.

**Comment by wlawt** [comment]
@youngkashew I think you may need to pull upstream one more time 😅

**Review by richardpringle — APPROVED** [review]

**Pull request merged**